### ISSUE TITLE
arrow announced by jaws

### ISSUE THREAD

**Issue by stevefaulkner** [ThePacielloGroup/disclosure-button#1]
JAWS 15 /firefox latest on windows 8.1 announces down arrow as 'down pointing black arrow' does not announce right arrow, should not announce either as state is provided via aria-expanded.

**Comment by stevefaulkner** [comment]
put aria-hidden true on icon span didn't make a difference - note JAWS announces down arrow in firefox only
